app/main/posts/views/filters/filter-saved-search.html

```diff
@@ -8,6 +8,7 @@
                         search in searches
                         track by search.id"
         >
+            <option value="" translate="app.select_one">Select one</option>
```

Comment:
I just checked and this does not revert back the search after being set since it's not bound to anything.